actionpack/lib/action_dispatch/middleware/executor.rb

```diff
@@ -9,6 +9,7 @@ def initialize(app, executor)
     def call(env)
       state = @executor.run!
       begin
+        puts "running"
```

Comment:
@matthewd because the `complete!` is called after the body proxy below is closed, my intercepter's "intercept" is always executed before the body is closed, not after.

So I can't check that `Current.reset` was called. I'm stumped 😕